freeCodeCamp/fcctesting.js

```diff
@@ -38,15 +38,15 @@
 const fs = require('fs');
 const cors = require('cors');
 
+const allowedOriginsMatcher = /^https?:\/\/([\w-]+\.)*freecodecamp.org/;
```

Comment:
Currently this regex will allow origins such as `https://www.freecodecamp-org.evil`

Changing it to ` /^https?:\/\/([\w-]+\.)*freecodecamp\.org/`  will enforce a `.` after `freecodecamp`